docs/how-to/nat-configuration.md

```diff
@@ -1,32 +1,136 @@
+
+## Configuration Options
 
-## IPv6
+The appropriate configuration option for your router depends on your specific setup:
+
+- If your router supports them, [enable IPv6](#enable-ipv6) or [enable UPnP](#enable-upnp) to solve most connection issues
+- If IPv6 or UPnP are not available, [enable manual port forwarding](#enable-manual-port-forwarding)
+
+### Enable IPv6
 
 If your router and internet service provider (ISP) support IPv6, enabling it will mitigate some connection issues. Most modern routers have a single option to enable IPv6. We are unable to give detailed information about each router's settings and preferences here. Search your router manufacturer's website for _IPv6_ for information on how to enable it.
 
-## UPnP
+### Enable UPnP
 
 If your router supports UPnP, IPFS will attempt to automatically allow inbound traffic to access your local content. Some home routers may need to be configured to explicitly enable UPnP. We are unable to give detailed information about each router's settings and preferences here. Search your router manufacturer's website for _UPnP_.
 
-## Port forwarding
+<!--
+### Enable DCUtR Holepunching 
+
+Not sure what to put here
+
+Is there a recommended repo or libp2p docs link that we can link to here that explains how to set this up for Kubo? 
+
+https://github.com/libp2p/specs/blob/master/relay/DCUtR.md doesn't seem right
+
+https://docs.libp2p.io/concepts/nat/hole-punching/#phase-ii-hole-punching? seems to be mostly conceptual
+
+https://blog.ipfs.tech/2022-01-20-libp2p-hole-punching/ seems to just explain the theory
+
+Maybe https://docs.rs/libp2p/latest/libp2p/tutorials/hole_punching/index.html and tell the reader that they can follow this tutorial to set up DCUtR Holepunching using Rust? Not specific to Kubo but seems like its the best we've got right now?
+!-->
+
+
+### Enable manual port forwarding
+
+If your router does not support UPNP and/or IPv6, or you want better reliability and performance than what DCUtR provides, set up manual port forwarding. Complete the following steps to enable manual port forwarding:
+
+1. [Open a port from the internet to your internal Kubo node](#open-a-port). 
+1. [Update the Kubo Configuration](#update-the-kubo-configuration).
+1. [Restart your Kubo node](#restart-your-kubo-node). 
 
-Each router has different options and solutions for port forwarding. Most router manufacturers have guides for setting up port forwarding on their devices. In general, the steps are something like this:
+First, open a port from the internet to your internal Kubo node.
+
+#### Open a port 
+
+:::tip
+Assuming you're not trying to expose your daemon's API to the public internet, opening port `4001`/`tcp` should be sufficient.
```

Comment:
As a user, this sentence lead me to belive exposing the api port could be something i could want to do, which no, never, dont do that.